src/app/user/user-login/user-login.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../user.service';
+import { NgForm } from '@angular/forms';
+
+@Component({
+  selector: 'app-user-login',
+  templateUrl: './user-login.component.html',
+  styleUrls: ['./user-login.component.css']
+})
+export class UserLoginComponent implements OnInit {
+
+  constructor( private userService:UserService ) { }
+
```

Comment:
U r using minAttrLength but i cant see it here ;-)

Reply:
It's invisible.